ivy/container/elementwise.py

```diff
@@ -1485,7 +1485,7 @@ def bitwise_left_shift(
 
     @staticmethod
     def static_bitwise_invert(
-        x: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        x: ivy.Container,
```

Comment:
This shouldn't have been changed in the process, only the docstrings were to be added as the typehints are correct, we take in `ivy.Array and ivy.NativeArray` as well in the static method . 🙂

Reply:
@zaeemansari70 I have tried this method out previously and it didn't work with arrays or native arrays, only with containers. That's why I have changed the type hints